Makefile

```diff
@@ -456,7 +456,162 @@ deploy-olm: undeploy-olm ## Build current branch operator image, bundle image, p
+	echo "https://$$CONSOLE_URL/operatorhub/ns/$(OADP_TEST_NAMESPACE)?keyword=oadp-operator&details-item=oadp-operator-oadp-operator-catalog-$(OADP_TEST_NAMESPACE)&channel=operator-sdk-run-bundle&version=$(VERSION)"
+	@echo ""
+	@echo "==========================================================================="
+
+.PHONY: deploy-olm-stsflow-gcp
+deploy-olm-stsflow-gcp: deploy-olm-stsflow ## Deploy via OLM with GCP WIF standardized flow and create subscription with GCP env vars
+	@if [ -n "$(GCP_PROJECT_NUM)" ] && [ -n "$(GCP_POOL_ID)" ] && [ -n "$(GCP_PROVIDER_ID)" ] && [ -n "$(GCP_SA_EMAIL)" ]; then \
+		echo "Creating subscription with GCP WIF environment variables..."; \
+		$(call create-sts-subscription,/tmp/oadp-gcp-subscription.yaml); \
+		echo "    - name: PROJECT_NUMBER" >> /tmp/oadp-gcp-subscription.yaml; \
+		echo "      value: \"$(GCP_PROJECT_NUM)\"" >> /tmp/oadp-gcp-subscription.yaml; \
+		echo "    - name: POOL_ID" >> /tmp/oadp-gcp-subscription.yaml; \
+		echo "      value: \"$(GCP_POOL_ID)\"" >> /tmp/oadp-gcp-subscription.yaml; \
+		echo "    - name: PROVIDER_ID" >> /tmp/oadp-gcp-subscription.yaml; \
+		echo "      value: \"$(GCP_PROVIDER_ID)\"" >> /tmp/oadp-gcp-subscription.yaml; \
+		echo "    - name: SERVICE_ACCOUNT_EMAIL" >> /tmp/oadp-gcp-subscription.yaml; \
+		echo "      value: \"$(GCP_SA_EMAIL)\"" >> /tmp/oadp-gcp-subscription.yaml; \
+		$(call apply-sts-subscription,/tmp/oadp-gcp-subscription.yaml,GCP WIF); \
+	else \
+		echo ""; \
+		echo "GCP WIF environment variables not set. Please set all of the following:"; \
+		echo "  GCP_PROJECT_NUM"; \
+		echo "  GCP_POOL_ID"; \
+		echo "  GCP_PROVIDER_ID"; \
+		echo "  GCP_SA_EMAIL"; \
+		echo ""; \
+		echo "Example:"; \
+		echo "  make deploy-olm-stsflow-gcp GCP_PROJECT_NUM=123456789 GCP_POOL_ID=my-pool GCP_PROVIDER_ID=my-provider GCP_SA_EMAIL=my-sa@my-project.iam.gserviceaccount.com"; \
+	fi
+
+.PHONY: deploy-olm-stsflow-aws
+deploy-olm-stsflow-aws: deploy-olm-stsflow ## Deploy via OLM with AWS STS standardized flow and create subscription with AWS env vars
+	@if [ -n "$(AWS_ROLE_ARN)" ]; then \
+		echo "Creating subscription with AWS STS environment variables..."; \
+		$(call create-sts-subscription,/tmp/oadp-aws-subscription.yaml); \
+		echo "    - name: ROLEARN" >> /tmp/oadp-aws-subscription.yaml; \
+		echo "      value: \"$(AWS_ROLE_ARN)\"" >> /tmp/oadp-aws-subscription.yaml; \
+		$(call apply-sts-subscription,/tmp/oadp-aws-subscription.yaml,AWS STS); \
+	else \
+		echo ""; \
+		echo "AWS STS environment variable not set. Please set:"; \
+		echo "  AWS_ROLE_ARN"; \
+		echo ""; \
+		echo "Example:"; \
+		echo "  make deploy-olm-stsflow-aws AWS_ROLE_ARN=arn:aws:iam::123456789012:role/my-oadp-role"; \
+	fi
+
+.PHONY: deploy-olm-stsflow-azure
+deploy-olm-stsflow-azure: deploy-olm-stsflow ## Deploy via OLM with Azure Workload Identity standardized flow and create subscription with Azure env vars
```

Comment:
Currently [blocked](https://redhat-internal.slack.com/archives/C02EL5R9C3B/p1749361497483569) from verifying azure.

Reply:
unblocked, got cluster.. although hit couple bumps along the way
https://issues.redhat.com/browse/CCO-687
https://issues.redhat.com/browse/CCO-685